src/invest_reports/sdr_ndr_utils.py

```diff
@@ -24,9 +27,23 @@ def build_intermediate_output_raster_plot_configs(
             for (output_id, datatype) in raster_plot_tuples]
 
 
-def generate_watershed_results_table(model_id, file_registry):
-    ws_vector_id = f'watershed_results_{model_id}'
-    ws_vector = geopandas.read_file(file_registry[ws_vector_id])
-    ws_vector_table = ws_vector.drop(columns=['geometry']).to_html(
-        index=False, na_rep='')
-    return ws_vector_table
+def generate_results_table_from_vector(filepath, cols_to_sum):
+    vector_df = geopandas.read_file(filepath)
+    vector_df = vector_df.drop(columns=['geometry'])
+
+    css_classes = ['datatable']
+    (num_rows, _) = vector_df.shape
```

Comment:
I don't think these parens are needed; could also index into `df.shape[0]` if you prefer.